go/stellar/stellar.go

```diff
@@ -990,7 +1002,8 @@ func ChangeAccountName(m libkb.MetaContext, accountID stellar1.AccountID, newNam
 			// Change Name in place to modify Account struct.
 			nextBundle.Accounts[i].Name = newName
 			found = true
-			break
+		} else if acc.Name == newName {
+			return fmt.Errorf("that account name is already taken")
```

Comment:
maybe "you already have an account with that name" ? so we don't imply that names are unique across the entire network

Reply:
oh that's weird. we already return that error from `ValidateAccountNameLocal`. maybe nevermind then

Reply:
I like yours better, will change both